pkg/backup/backup_job.go

```diff
@@ -599,9 +567,12 @@ func (b *backupResumer) Resume(ctx context.Context, execCtx interface{}) error {
 	// Resolve the backup destination. We can skip this step if we
 	// have already resolved and persisted the destination either
 	// during a previous resumption of this job.
-	defaultURI := details.URI
+	var backupManifest *backuppb.BackupManifest
 	var backupDest backupdest.ResolvedDestination
-	if details.URI == "" {
+	details := initialDetails
+	needsInitializing := initialDetails.URI == ""
+	if needsInitializing {
+		var err error
```

Comment:
i don't think this local var err is necessary anymore once backupDest is created below.

Reply:
Another artifact from the larger PR which used it.